modules/dnn/src/layers/prior_box_layer.cpp

```diff
@@ -126,6 +126,36 @@ class PriorBoxLayerImpl CV_FINAL : public PriorBoxLayer
         }
     }
 
+    void getMinSize(const LayerParams &params)
+    {
+        DictValue minSizeParameter;
+        bool minSizeRetieved = getParameterDict(params, "min_size", minSizeParameter);
+        if (!minSizeRetieved) {
+            _minSize.push_back(0);
+            return;
+        }
+
+        for (int i = 0; i < minSizeParameter.size(); ++i)
+        {
+            float min = minSizeParameter.get<float>(i);
+            _minSize.push_back(min);
+        }
+    }
+
+    void getMaxSize(const LayerParams &params)
+    {
+        DictValue maxSizeParameter;
+        bool maxSizeRetieved = getParameterDict(params, "max_size", maxSizeParameter);
+        if (!maxSizeRetieved)
+            return;
+
+        for (int i = 0; i < maxSizeParameter.size(); ++i)
+        {
+            float maxSize = maxSizeParameter.get<float>(i);
+            _maxSize.push_back(maxSize);
+        }
+    }
```

Comment:
There is `static void getParams(const std::string& name, const LayerParams &params, std::vector<float>* values)`. Can we reuse it?